elements/rh-pagination/demo/right-to-left.html

```diff
@@ -0,0 +1,17 @@
+<div dir="rtl">
+  <p>צריך להיראות יותר טוב</p>
+  <rh-pagination>
+  <span slot="go-to-page">עבור לדף</span>
```

Comment:
Because there's more than one child, I'm seeing this validation error in the deployment preview: https://github.com/RedHat-UX/red-hat-design-system/pull/1082/files#diff-f92ce41450abc890f2465fb02e478b48cc6cfcc66d9aeb4279f5091e282438ffR237

Reply:
Should fix that in a separate PR, because this is legit usage

Reply:
Cool, cool. Thx